lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -69,6 +69,29 @@ module Resources
       'relationship': [
         {
           'id': 'CI-490017058-0-0',
+          'extension': [
+            {
+              'valueCodeableconcept': {
+                'coding': [
+                  {
+                    'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+                    'code': 'SIS',
+                    'display': 'sister',
+                    'userSelected': false
+                  }
+                ],
+                'text': 'Sister'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relation'
+            },
```

Comment:
we should be having the proprietary codes for the relation extension with codeset 40

Reply:
added here and in other places as well
https://github.com/cerner/fhir.cerner.com/pull/624/commits/7532319ad69a4ed7dc8e889132d3547cbd62f59c#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR84